harvester/tests
# fix daily e2e errors

### Changes
- [x] Update `api_endpoints.json.j2` to fix test cases in **harvester_e2e_tests/apis/test_vm_templates.py**
- [x] Fix test scenario: most available CPU
    - Update `basic_vm.json.j2` to inject **CPU** correctly
    - Update assert **available_cpu** to `1`, because we will expect that the created VM will initial with `1` CPU
    - Implement `parse_unit` to handle resource size
    - Update `lookup_hosts_with_most_available_cpu` to use **Reserved** resource instead of **Used**
- [x] Fix test scenario: most available Memory
    - Update assert **available_memory** to `1`, because we will expect that the created VM will initial with `1` Memory
    - Implement `format_unit` to handle resource size convert
    - Update `lookup_hosts_with_most_available_memory` to use **Reserved** resource instead of **Used**, to avoid over-commit, also reduce maximum memory to _99%_ before floor the size.
